hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java

```diff
@@ -476,7 +475,7 @@ protected void validateAdapterSchema(
             case NOT_FOUND:
                 return ErrorResponseUtil.notFound("Tag", decodedTagName);
         }
-        return Response.serverError().build();
+        return Response.serverError().entity(INTERNAL_ERROR_PLEASE_CHECK_LOGS).build();
```

Comment:
we should log in this case as it is a fall through and we want to know the enum value that caused that.

Reply:
Done.